Cargo.toml

```diff
@@ -42,7 +42,7 @@ num-derive = "0.4"
 num-traits = "0.2"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
-serde_yaml = "0.9.25"
+serde_yaml = "0.9.23"
```

Comment:
why do we need to downgrade here?

Reply:
sorry, that was a leftover while trying to bisect the breaking change